### PR TITLE
fix FOSUserBundle.nl.yml reset password email subject

### PR DESCRIPTION
**Pull request by fender-hm**

### CODE REVIEW

File: Resources/translations/FOSUserBundle.nl.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ resetting:
     flash:
         success: Het wachtwoord is succesvol gereset
     email:
-        subject: Welkom %username%!
+        subject: Reset Password
```
Comment (sstok) [inline Resources/translations/FOSUserBundle.nl.yml]: Password -> Wachtwoord ;)

**Comment by stof** [comment]
@fender-hm can you update your PR ?

**Comment by stof** [comment]
@fender-hm ping

**Comment by fender-hm** [comment]
@stof hello, sorry that so long withut a reaction. PR updated.

**Comment by sstok** [comment]
:+1:

**Comment by OskarStark** [comment]
:+1:

**Comment by XWB** [comment]
Fixed in https://github.com/FriendsOfSymfony/FOSUserBundle/commit/45db1e7930cf5728336bb3bd3258b8eb5094c50f

**Pull request closed without merging**